platform/ui/src/components/StudyBrowser/StudyBrowser.tsx

```diff
@@ -84,8 +84,10 @@ const StudyBrowser = ({
             return (
               <Button
                 key={name}
+                name={name}
```

Comment:
Specify the name so it can be found and the text color as white so it is visible.